src/commands/shell.rs

```diff
@@ -73,20 +93,135 @@ pub async fn command(args: Args, _json: bool) -> Result<()> {
         eprintln!("No service linked, skipping service variables");
     }
 
+    // disable dead code warning for windows_shell_detection
+    #[allow(dead_code)]
+    enum WindowsShell {
+        Cmd,
+        Powershell,
+        Powershell7,
+    }
+
+    /// https://gist.github.com/mattn/253013/d47b90159cf8ffa4d92448614b748aa1d235ebe4
+    /// Only matches cmd, powershell or pwsh for safety
```

Comment:
Consider adding:
- nushell
- elv.sh

Reply:
Unable to test until later tonight. Will commit then.

Thanks for the feedback.

Reply:
Added nushell and elv.sh